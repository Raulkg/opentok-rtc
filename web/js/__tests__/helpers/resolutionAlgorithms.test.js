var resolutionAlgorithms = require('../../helpers/resolutionAlgorithms.js');
test('snapshot for resolutionAlgorithms component', () => {
  expect(resolutionAlgorithms).toMatchSnapshot();
});