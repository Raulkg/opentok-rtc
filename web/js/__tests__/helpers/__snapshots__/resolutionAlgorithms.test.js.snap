// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot for resolutionAlgorithms component 1`] = `
Object {
  "PreferredResolutionAlgorithmProvider": Object {
    "algorithmNames": Array [
      "percentOfAvailable",
      "percentOfStream",
      "biasedPercent",
      "fitToSubscriberDimensions",
    ],
    "defaultAlgorithmName": "fitToSubscriberDimensions",
    "getAlg": [Function],
  },
}
`;
